src/store/reducers/list.js

```diff
@@ -1,5 +1,5 @@
-import {ADD_TASK, REMOVE_TASK} from '../variables';
-const initialState = { tasks: ['First task'] };
+import {ADD_TASK, REMOVE_TASK, SAVE} from '../variables';
+const initialState = JSON.parse(localStorage.getItem('list')) || { tasks: [] };
```

Comment:
initial state should `{ tasks: []}`, taking from localStorage should be handled via `componentDidMount`